fix wifi account setup procedure text (#1074)
utelecon/utelecon.github.io: src/components/en/systems/utokyo_wifi/index.mdx

  <Fragment slot="else">
    ### Step 1: Check the devices to be used

      Laptops, smartphones, and tablets can generally connect to UTokyo Wi-Fi. However, other types of electronic devices and older devices may not be able to connect. For example, devices that do not support WPA2-Enterprise cannot be connected. Please check for updates to the OS and drivers of your devices and update them appropriately before connecting to UTokyo Wi-Fi.

    ### Step 2: Apply for a UTokyo Wi-Fi Account

>       UTokyo Wi-Fi is available to UTokyo Account holders, but UTokyo Wi-Fi Account is required for each academic year in addition to your UTokyo Account. In this “Step 2”, we will explain the procedure for applying for a dedicated UTokyo Wi-Fi account.

Reviewer: ```suggestion
       In order to use UTokyo Wi-Fi, a UTokyo Wi-Fi account must be issued each academic year. In this “Step 2”, we will explain the procedure for applying for a UTokyo Wi-Fi account.
```
